.npmignore

```diff
@@ -0,0 +1,39 @@
+# File System
```

Comment:
Why do we have all this files here? Should they be in gitignore file? They will not exist in the jenkins machine.

Reply:
We use it across all npm packages.